README.md

```diff
@@ -265,6 +266,18 @@ android {
 }
 ```
 
+### Enable Detekt checks only on changed files
+The plugin provides the ability to check only changed files. You can enable this:
+```kotlin
+redmadrobot {
+    detekt {
+        checkOnlyDiffWithBranch("develop")
+    }
+}
+```
+The plugin then adds a `detektDiff` task that allows you to check only changed files versus the base branch. The modified files are provided by Git.
+> Task `detektDiff` work with rules without **type resolutions**. About **type resolution** you can read [here](https://detekt.github.io/detekt/type-resolution.html).
```

Comment:
It is better to leave blank lines between code blocs, callouts, headers, and text. Also, it is useful to break lines before each sentence.  
Also, here text can be shorter.

```suggestion
The plugin then adds a `detektDiff` task that allows you to check only changed files versus the base branch.
The modified files are provided by Git.

> Task `detektDiff` is not compatible with [**type resolution**](https://detekt.github.io/detekt/type-resolution.html).
> It means some of the configured rules will not work.
```
Btw maybe better to place the disclaimer about type resolution under detekt tasks list? Because all of the tasks not work with type resolution.

Reply:
Formatted text and added a note to the `detekt` section.